### PR TITLE
Set up pdf reader API

### PR DESCRIPTION
**Pull request by daniel19e**
Created a flask API endpoint that gets a pdf file from the user's computer and extracts all the text. This is the first step for the note generation process.

### CODE REVIEW

File: .DS_Store
Comment (coderpotter) [inline .DS_Store:1]: put this in `.gitignore`

File: __pycache__/app.cpython-310.pyc
Comment (coderpotter) [inline __pycache__/app.cpython-310.pyc:1]: put this in `.gitignore`

File: __pycache__/readpdf.cpython-310.pyc
Comment (coderpotter) [inline __pycache__/readpdf.cpython-310.pyc:1]: put this in `.gitignore`

File: app.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+app.register_blueprint(api_blueprint, url_prefix="/api")
+
+
+@app.route("/")
+def index():
+    return render_template("upload.html")
+
+
+if __name__ == "__main__":
```
Comment (coderpotter) [inline app.py:14]: Having this is important. Good job on following good code practices. Keep it up.

**Review by coderpotter — COMMENTED** [review]
I’ll neither approve nor reject PRs because I’m just in an advisory position and don’t know about your entire S/W engineering pipeline. However, I’ll write comments for you to address if you see fit.

**Comment by coderpotter** [comment]
Are you planning to support URLs too, or just locally stored PDFs?

**Comment by daniel19e** [comment]
> Are you planning to support URLs too, or just locally stored PDFs?

We will add support for URLs in another PR.

**Pull request merged**